Feedback/templates/Feedback/register.html

```diff
@@ -0,0 +1,70 @@
+{%extends 'base.html'%}
```

Comment:
Not a big issue. But I prefer to give space after "{% " and before " %}" in Django tags.